nemo/lightning/resume.py

```diff
@@ -244,15 +244,15 @@ def get_context_path(self, model: Optional[io.ConnectorMixin] = None) -> Optiona
             checkpoint = self._find_trainer_ckpt_path()
 
         if checkpoint:
-            maybe_model_weights_path = Path(checkpoint) / "context"
-            if os.path.isdir(maybe_model_weights_path):
-                checkpoint = maybe_model_weights_path
+            maybe_context_path = Path(checkpoint) / "context"
+            if maybe_context_path.is_dir():
+                checkpoint = maybe_context_path
         return checkpoint
 
     def get_trainer_ckpt_path(self, model: Optional[io.ConnectorMixin] = None) -> Optional[Path]:
         if self.resume_from_path:
-            maybe_model_weights_path = self.get_model_weights_path(self.resume_from_path)
-            return maybe_model_weights_path if os.path.isdir(maybe_model_weights_path) else self.resume_from_path
+            maybe_weights_path = self.get_weights_path(self.resume_from_path)
+            return maybe_weights_path if os.path.isdir(maybe_weights_path) else self.resume_from_path
```

Comment:
should this be `maybe_weights_path.is_dir()`?

Reply:
Yes, thanks!